packages/assets-controllers/src/TokenBalancesController.ts

```diff
@@ -464,6 +475,16 @@ export class TokenBalancesController extends StaticIntervalPollingController<{
     );
   }
 
+  /**
+   * Get the default polling interval for this controller
+   * This is the base interval used for chains without specific configuration
+   * 
+   * @returns The default polling interval in milliseconds
+   */
+  getDefaultPollingInterval(): number {
```

Comment:
It looks like we are adding a new method to the controller, should we mention this in the changelog?